sdcm/cluster_k8s/__init__.py

```diff
@@ -1349,6 +1349,7 @@ def wait_for_k8s_node_readiness(self):
         wait_for(self._wait_for_k8s_node_readiness,
                  text=f"Wait for k8s host {self.node_name} to be ready...",
                  timeout=self.pod_readiness_timeout * 60,
+                 step=5,
```

Comment:
Why do we need it? _wait_for_k8s_node_readiness is already waiting significant amount of time

Reply:
Default step/interval is 1 sec and it is too fast for such thing which takes minutes. So, I reduced it to be 5 seconds to have fewer attempts and fewer "waiting" messages in SCT log

Reply:
Yes, but _wait_for_k8s_node_readiness is already waiting for pod_readiness_timeout//3, which is 10 minutes for scylla pods, what difference would 5 seconds do here ?

Reply:
"_wait_for_k8s_node_readiness" fails fast and we do such call each 1 sec. So, it doesn't matter at all what we have in "_wait_for_k8s_node_readiness" in such a case. That's why "step" value for "wait_for" really matters here.
Increasing "step" we reduce number of redundant API calls and SCT log spam.

Reply:
agree, lets keep it